sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -124,13 +124,33 @@ class ApiManager {
   static readonly terminalApiUrl = '/api/terminals';
```

Comment:
We should remove all `/` prefix from the api urls here, except for `/api/basepath` right?